docs/index.rst

```diff
@@ -114,6 +114,14 @@
 .. toctree::
   :maxdepth: 0
   :hidden:
+  :caption: Stackdriver Error Reporting
```

Comment:
Public branding is Stackdriver Monitoring, Stackdriver Logging, etc. Cloud Logging/Cloud Monitoring are not right, I was going to make a PR to fix this all in the other places. I don't necessarily understand this branding direction but it's definitely what our branding guidline says.